src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -212,8 +218,31 @@ ss::future<> service::configure() {
           principal,
           security::acl_host::wildcard_host(),
           security::acl_operation::all,
-          security::acl_permission::allow}}},
-      5s);
+          security::acl_permission::allow}}};
+
+    auto create_acls_stop_time = model::time_from_now(30s);
+    while (err_vec[0] != cluster::errc::success
+           && ss::lowres_clock::now() < create_acls_stop_time) {
+        err_vec
+          = co_await _controller->get_security_frontend().local().create_acls(
+            princpal_acl_binding, 1s);
+
+        if (err_vec[0] != cluster::errc::success) {
+            co_await ss::sleep_abortable(create_acls_backoff, shard0_as);
+            create_acls_backoff *= 2;
```

Comment:
I think this should be limited to 1s or something. Otherwise the last (~10th) wait is likely to be 10s or so, which is unnecessarily large.